app/main.py

```diff
@@ -14,12 +14,38 @@
 from sdx.common.logger_config import logger_initial_config
 import tornado.ioloop
 import tornado.web
+from tornado.httpclient import HTTPClient, HTTPError
 
 from encrypter import Encrypter
 from ftpclient import FTPWorker
 from publisher import DurableTopicPublisher
 
 
+class HealthCheckService(tornado.web.RequestHandler):
+
+    def initialize(self, services):
+        self.ftp = FTPWorker(**Task.ftp_params(services))
+        self.amqp_params = Task.amqp_params(services)
+
+    def get(self):
+        http_client = HTTPClient()
+        try:
+            http_client.fetch(self.ampq_params["check"])
```

Comment:
We decided to add this call to a background task instead of running every time the endpoint is hit. This call could also block as you are not using the AsyncHTTPClient.